docs/advanced.rst

```diff
@@ -1019,104 +1143,651 @@ to learn more about the use of this feature.
+| **XML**                                         |
++-------------------------------------------------+
+| .. literalinclude:: ../code/StaticTester.xml    |
+|    :language: xml                               |
+|    :start-after: <!-->STATIC_DISCOVERY_CONF<--> |
+|    :end-before: <!--><-->                       |
++-------------------------------------------------+
+
+.. _`static_xml_load`:
+
+Loading STATIC EDP XML Files
+----------------------------
+
+Statically discovered remote endpoints **must** define a unique *userID* on their profile, which value **must** agree
+with the one specified on the discovery configuration XML.
+This is done by setting the user ID on the entity attributes:
+
++--------------------------------------------------------+
+| **C++**                                                |
++--------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp             |
+|    :language: c++                                      |
+|    :start-after: //CONF_QOS_STATIC_DISCOVERY_USERID    |
+|    :end-before: //!                                    |
++--------------------------------------------------------+
+| **XML**                                                |
++--------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml              |
+|    :language: xml                                      |
+|    :start-after: <!-->CONF_QOS_STATIC_DISCOVERY_USERID |
+|    :end-before: <!-->                                  |
++--------------------------------------------------------+
+
+On the local participant, loading STATIC EDP configuration files is done by:
+
++---------------------------------------------------------+
+| **C++**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp              |
+|    :language: c++                                       |
+|    :start-after: //CONF_STATIC_DISCOVERY_XML            |
+|    :end-before: //!                                     |
++---------------------------------------------------------+
+| **XML**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml               |
+|    :language: xml                                       |
+|    :start-after: <!-->CONF_STATIC_DISCOVERY_XML         |
+|    :end-before: <!-->                                   |
++---------------------------------------------------------+
+
+.. role:: raw-html(raw)
+    :format: html
+
+.. _discovery_server:
+
+Server-Client Discovery
+=======================
+
+This mechanism is based on a client-server discovery paradigm, i.e. the metatraffic (message exchange among participants
+to identify each other) is centralized in one or several server participants (left figure), as opposed to simple
```

Comment:
in one or several participants -> maybe I would use 'through' in spite of 'in'?

Reply:
After thinking a while about this I've decided that rephrasing is the path to go:

> The metatraffic is managed by one or several server participants…

because say centralized when there may be many *centers* doesn't make much sense.